.github/workflows/async-upload-test.yml

```diff
@@ -17,11 +17,7 @@ on:
       - ".github/workflows/**"
 
 env:
-  IMG_REGISTRY: ghcr.io
-  IMG_ORG: kubeflow
```

Comment:
To ease maintenance in m/s i would really prefer to have these at the top, this way we just adjust OCI reference in a single place when porting. Can you restore these Envs here, please?

Reply:
Yeah, there are some issues with how the ENV vars are being passed around to downstream make commands, etc. So I think a more comprehensive refactor of all the make files will be needed. This was really done to get all the tests to actually use the variables it should be using. For example, even providing an `IMG` to the make file for the mr-server image will not always take the correct `IMG_VERSION`. There are some hard-coded instances in some of these makefiles.

Let me take a look and see if I can do a minimal refactor with the env vars here restored